.circleci/config.yml

```
@@ -35,7 +35,7 @@ jobs:
 
             - run:
                 name: Install Python2
-                command: sudo apt-get update && sudo apt-get install -y python2
+                command: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B && sudo apt-get update && sudo apt-get install -y python2
```

Does this work if it's split into two lines?
```suggestion
                command: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B
                command: sudo apt-get update && sudo apt-get install -y python2
```

---

Wait, no, that won't work.